src/main/kotlin/uk/gov/dwp/dataworks/aws/AwsCommunicator.kt

```diff
@@ -489,4 +490,8 @@ class AwsCommunicator {
                 "correlation_id" to correlationId,
                 "user_name" to primaryKeyValue)
     }
+
+    fun getAccNumber(): String? {
```

Comment:
Can we make this safe please? Ideally we should avoid nullable types when we can

Reply:
I wasn't involved in the discussion to make the ARNs predictable - this is what Casey recommended, after Ian and him decided on this new way of allowing access. I will talk with them and find out if it is returning the account details that we need.